common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -178,6 +179,142 @@ def export(self):
+                                <lomimscc:identifier>
+                                </lomimscc:identifier>
+                                <lomimscc:keyword>
+                                </lomimscc:keyword>
+                            </lomimscc:general>
+                        </lomimscc:lom>
+                    </metadata>
+                </item>
+                """
+
+    def __resource_entry(self):
+        return """<resource xmlns="http://www.imsglobal.org/xsd/imsccv1p2/imscp_v1p1"
+                    xmlns:lom="http://ltsc.ieee.org/xsd/imsccv1p2/LOM/resource"
+                    xmlns:csm="http://www.imsglobal.org/xsd/imsccv1p2/imscsmd_v1p0"
+                identifier="{counter}_R" type="imsbasiclti_xmlv1p0">
+              <file href="{filename}"/>
+              <metadata></metadata>
+            </resource>
+            """
+
+    def __resource_file_entry(self):
+        return """<?xml version="1.0" encoding="UTF-8"?>
+            <cartridge_basiclti_link
+                xmlns="http://www.imsglobal.org/xsd/imslticc_v1p0"
+                xmlns:blti="http://www.imsglobal.org/xsd/imsbasiclti_v1p0"
+                xmlns:lticm="http://www.imsglobal.org/xsd/imslticm_v1p0"
+                xmlns:lticp="http://www.imsglobal.org/xsd/imslticp_v1p0"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://www.imsglobal.org/xsd/imslticc_v1p0
+                http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticc_v1p0p1.xsd
+                http://www.imsglobal.org/xsd/imsbasiclti_v1p0
+                http://www.imsglobal.org/xsd/lti/ltiv1p0/imsbasiclti_v1p0p1.xsd
+                http://www.imsglobal.org/xsd/imslticm_v1p0
+                http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticm_v1p0.xsd
+                http://www.imsglobal.org/xsd/imslticp_v1p0
+                http://www.imsglobal.org/xsd/lti/ltiv1p0/imslticp_v1p0.xsd" 
+                >    
+                <blti:title>{title}</blti:title>
+                <blti:secure_launch_url>{lti_link}</blti:secure_launch_url>
+            </cartridge_basiclti_link>"""
+
+    def export(self):
+        """
+        Perform the export given the parameters handed to this class at init.
+        """
+        with self.modulestore.bulk_operations(self.courselike_key):
+            export_fs = OSFS(self.root_dir)
+            xmlns_link = "http://www.imsglobal.org/xsd/imsccv1p2/imscp_v1p1"
+            format_num = "I_{:0>6}"
+            with export_fs.open('imsmanifest.xml', 'w') as course_xml:
+                minifest_doc = self.__imsmanifest_entry().format(title=self.courselike_key.course,
+                                                                 xmlns_link=xmlns_link)
+                tree = lxml.etree.ElementTree(element=None, file=StringIO(minifest_doc))
+                root = tree.getroot()
+                seqs = self.modulestore.get_course(self.courselike_key, depth=0).get_children()
```

Comment:
It looks like this expression return **chapters** instead of **sequential** blocks.
I may suggest to use such statement:
```
blocks = modulestore().get_items(course_key, qualifiers={'category': 'sequential'})
```